### ISSUE TITLE
[Usable]Name for 'Download icon' button is not properly defined.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1316]
**Actual Result:**
When focus lands on 'download icon' in scan mode then narrator is reading "Empty paragraph".
Name property is not properly defined for 'Download icons'.

**Note -** Same issue repro at -
1. URL -> Help -> Document Data URI button/File attachments Button -> Download button icon - When focus lands on 'download icon' forward navigation of scan mode then narrator is reading "Empty paragraph", narrator not reading the name for download icon.

**Expected Result:**
When focus lands on 'download icon' in scan mode then narrator should not read "Empty paragraph".
Name property should be properly defined for 'download icons'

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3.  Type 'accessibility' in the input control at the bottom of the screen and hit enter (or click on send button).
4.  Page will upload.
5. Open Narrator with scan mode (Caps+Space).
6. Navigate to "download icon" by using Up/Down Arrow keys.
7. Observe that Narrator is not reading name for "download icon" while navigate in the scan mode.

**User Impact:**
If Name property is not defined properly for control then the users will get confuse about the purpose of that control.

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
[Usable_Name for download icon.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2584552/Usable_Name.for.download.icon.zip)

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yUsable
